.github/CODEOWNERS

```diff
@@ -2,4 +2,8 @@
 # Each line is a file pattern followed by one or more owners.
 # Syntax can be found here: https://docs.github.com/free-pro-team@latest/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
 
-/src/sql/*.d.ts @alanrenmsft @Charles-Gagnon
+/src/sql/*.d.ts @alanrenmsft @Charles-Gagnon @ranasaria
+/src/extensions/resource-deployment/* @ranasaria
+/src/extensions/arc/ @ranasaria
```

Comment:
why are there different ways to match files here, resource-deployment has a trailing * but these do not, actually according to the syntax, you don't need that.

Reply:
good catch the * is not needed for resource-deployment as that will just match to files directly under resource-deployment

Reply:
Fixed.